src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReadsIntegrationTest.java

```diff
@@ -59,7 +59,7 @@ public void testSplitsFixNDN() throws Exception {
     @Test //regression test for https://github.com/broadinstitute/gatk/pull/1853
     public void testSplitsOfUnpairedAndUnmappedReads() throws Exception {
         IntegrationTestSpec spec = new IntegrationTestSpec(
-                "-R" + b37_reference_20_21 + " -I " + largeFileTestDir + "K-562.duplicateMarked.chr20.bam -O %s --process-secondary-alignments " +
+                "-R " + b37_reference_20_21 + " -I " + largeFileTestDir + "K-562.duplicateMarked.chr20.bam -O %s --process-secondary-alignments " +
```

Comment:
How did this test work before with no space between `-R` and the reference file?

Reply:
The `jopt` parser's option clustering code accepted `-Rhuman_g1k_v37.20.21.fasta` as `-R human_g1k_v37.20.21.fasta`, but only because it couldn't peel off any more options (a different reference file might have failed). But we disabled option clustering in Barclay 4.0 because it had , so it will no longer succeed.